src-plugins/manualRegistration/manualRegistrationToolBox.cpp

```diff
@@ -481,6 +503,11 @@ void manualRegistrationToolBox::disableSaveButtons(bool param)
     d->b_exportTransformation->setDisabled(param);
 }
 
+void manualRegistrationToolBox::disableComputeButton(bool param)
+{
+    d->b_computeRegistration->setDisabled(param);
+}
```

Comment:
Well, there is already a `disableSaveButtons(..)` function, so I wanted to keep an homogeneity between these kind of functions. And, if we want in a close future to add a condition to the button, it'll be way easier for dev to do that. I think it's good coding :)